sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentSentiment.cs

```diff
@@ -11,13 +11,15 @@ namespace Azure.AI.TextAnalytics
     /// <summary>
     /// Overall predicted sentiment and confidence scores for the document.
     /// It also includes per-sentence sentiment prediction.
+    /// For more information regarding text sentiment, see
+    /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-sentiment-analysis"/>.
```

Comment:
I remember talking about adding that this is a soft max score in the docstring. Did we decide just to link out to the docs instead?